css-ui-3/caret-color-016.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: caret-color dynamic changes</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#caret-color">
```

Comment:
Since you're testing the serialization of <color>, you should add a `<link ref="help"` to the spec that defines that (css-color-3?)